subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/StrictDependenciesIntegrationTest.groovy

```diff
@@ -19,7 +19,6 @@ import spock.lang.Unroll
 
 class StrictDependenciesIntegrationTest extends AbstractModuleDependencyResolveTest {
 
-    @Unroll
```

Comment:
Was this required to ensure the test was executing? I thought @melix fixed this?

Reply:
I figured we just forgot to remove it. It does not make a difference. So I removed it now.